fotoapparat/src/main/java/io/fotoapparat/hardware/v1/capabilities/CapabilitiesFactory.java

```diff
@@ -86,4 +88,52 @@ public Capabilities fromParameters(Camera.Parameters parameters) {
 		return result;
 	}
 
+	private DiscreetRange extractSupportsSensorSensitivityValue(Camera.Parameters parameters) {
+		final Set<Integer> isoValuesSet = new HashSet<>();
+
+		// Based on https://stackoverflow.com/a/23567103/791323
+		String flat = parameters.flatten();
+		String[] isoValues = null;
+		String values_keyword=null;
+		String iso_keyword=null;
+
+		if (flat == null) {
+			return new DiscreetRange<>(Collections.<Integer>emptySet());
+		}
+
+		if (flat.contains("iso-values")) {
+            // most used keywords
+            values_keyword = "iso-values";
+            iso_keyword = "iso";
+        } else if (flat.contains("iso-mode-values")) {
+            // google galaxy nexus keywords
```

Comment:
Avoid using comments in the code. Extract constants with self-descriptive names instead.